2018-06-30 03:54:12.770 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 03:54:12.773 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 03:55:12.770 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 03:55:12.774 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 03:56:12.771 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 03:56:12.774 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 03:56:33.693 20577 DEBUG oslo_concurrency.lockutils [req-d21fe6e2-5bb1-40ed-9615-68aaf1ffde6a a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:56:34.292 20577 DEBUG neutron.agent.dhcp.agent [req-9affdf67-6b39-4027-948c-5e8fc721ac40 - - - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 03:56:34.294 20577 DEBUG neutron.agent.linux.utils [req-9affdf67-6b39-4027-948c-5e8fc721ac40 - - - - -] Unable to access /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:56:34.296 20577 DEBUG neutron.agent.linux.dhcp [req-9affdf67-6b39-4027-948c-5e8fc721ac40 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-639aee43-8461-496d-9fcf-c6db58e2fad0 on network 639aee43-8461-496d-9fcf-c6db58e2fad0 does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1267
2018-06-30 03:56:34.297 20577 DEBUG neutron.agent.linux.dhcp [req-9affdf67-6b39-4027-948c-5e8fc721ac40 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-639aee43-8461-496d-9fcf-c6db58e2fad0 on network 639aee43-8461-496d-9fcf-c6db58e2fad0 does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1288
2018-06-30 03:56:37.561 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.584 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.600 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.622 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.638 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.656 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', 'link', 'set', 'tap7e2a3353-01', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.684 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-o', 'link', 'show', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.702 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-30 03:56:37.728 20577 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tap7e2a3353-01) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-30 03:56:37.729 20577 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-06-30 03:56:37.731 20577 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tap7e2a3353-01) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-30 03:56:37.733 20577 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'7e2a3353-0146-469e-946a-d5421e490afd', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:75:3c:79'})), record=tap7e2a3353-01) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-30 03:56:37.772 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap7e2a3353-01', 'address', 'fa:16:3e:75:3c:79'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.784 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.797 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap7e2a3353-01', 'netns', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.825 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', 'link', 'set', 'tap7e2a3353-01', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.842 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', 'link', 'set', 'tap7e2a3353-01', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.858 20577 DEBUG oslo_concurrency.lockutils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Acquired semaphore "iptables-qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:56:37.860 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.887 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.909 20577 DEBUG neutron.agent.linux.iptables_manager [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-06-30 03:56:37.911 20577 DEBUG oslo_concurrency.lockutils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Releasing semaphore "iptables-qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:56:37.912 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-o', 'link', 'show', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.928 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', 'addr', 'show', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.943 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'addr', 'add', '11.135.1.2/24', 'scope', 'global', 'dev', 'tap7e2a3353-01', 'brd', '11.135.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.958 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.973 20577 DEBUG neutron.agent.linux.dhcp [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Setting IPv4 gateway for dhcp netns on net 639aee43-8461-496d-9fcf-c6db58e2fad0 to 11.135.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1164
2018-06-30 03:56:37.975 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.135.1.1', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:37.989 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-6', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:38.005 20577 DEBUG neutron.agent.linux.dhcp [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Cleaning stale devices for network 639aee43-8461-496d-9fcf-c6db58e2fad0 _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1369
2018-06-30 03:56:38.006 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:38.019 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:38.038 20577 DEBUG neutron.agent.linux.dhcp [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Building initial lease file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:623
2018-06-30 03:56:38.040 20577 DEBUG neutron.agent.linux.dhcp [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Done building initial lease file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/leases with contents:
1530410198 fa:16:3e:75:3c:79 11.135.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:651
2018-06-30 03:56:38.040 20577 DEBUG neutron.agent.linux.dhcp [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Building host file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 03:56:38.042 20577 DEBUG neutron.agent.linux.dhcp [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Done building host file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 03:56:38.045 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Unable to access /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:56:38.047 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host', '--addn-hosts=/var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tap7e2a3353-01', '--dhcp-range=set:tag0,11.135.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:38.083 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:56:38.085 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:56:38.087 20577 DEBUG neutron.agent.linux.external_process [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 03:56:38.089 20577 DEBUG oslo_concurrency.lockutils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:56:38.091 20577 DEBUG oslo_concurrency.lockutils [req-d21fe6e2-5bb1-40ed-9615-68aaf1ffde6a a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:56:38.691 20577 DEBUG neutron.agent.dhcp.agent [req-d7b4f5ca-6cd3-4ee2-bb9d-6757e6ff6546 - - - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 03:56:38.693 20577 DEBUG neutron.agent.linux.dhcp [req-d7b4f5ca-6cd3-4ee2-bb9d-6757e6ff6546 - - - - -] Building host file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 03:56:38.698 20577 DEBUG neutron.agent.linux.dhcp [req-d7b4f5ca-6cd3-4ee2-bb9d-6757e6ff6546 - - - - -] Done building host file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 03:56:38.703 20577 DEBUG neutron.agent.linux.utils [req-d7b4f5ca-6cd3-4ee2-bb9d-6757e6ff6546 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '29046'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:38.719 20577 DEBUG neutron.agent.linux.dhcp [req-d7b4f5ca-6cd3-4ee2-bb9d-6757e6ff6546 - - - - -] Reloading allocations for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 03:56:38.722 20577 DEBUG neutron.agent.linux.utils [req-d7b4f5ca-6cd3-4ee2-bb9d-6757e6ff6546 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:38.740 20577 DEBUG neutron.agent.linux.utils [req-d7b4f5ca-6cd3-4ee2-bb9d-6757e6ff6546 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-6', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:38.757 20577 DEBUG neutron.agent.linux.utils [req-d7b4f5ca-6cd3-4ee2-bb9d-6757e6ff6546 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:56:38.760 20577 DEBUG neutron.agent.linux.utils [req-d7b4f5ca-6cd3-4ee2-bb9d-6757e6ff6546 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:56:38.761 20577 DEBUG neutron.agent.linux.external_process [req-d7b4f5ca-6cd3-4ee2-bb9d-6757e6ff6546 - - - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 03:56:38.762 20577 DEBUG oslo_concurrency.lockutils [req-d7b4f5ca-6cd3-4ee2-bb9d-6757e6ff6546 - - - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:56:38.765 20577 DEBUG oslo_concurrency.lockutils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:56:38.766 20577 DEBUG neutron.agent.dhcp.agent [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-30T01:56:34Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-639aee43-8461-496d-9fcf-c6db58e2fad0, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4f3a9de3-0882-42a8-b7e9-2407f70f728a', u'ip_address': u'11.135.1.2'}], id=7e2a3353-0146-469e-946a-d5421e490afd, mac_address=fa:16:3e:75:3c:79, name=, network_id=639aee43-8461-496d-9fcf-c6db58e2fad0, port_security_enabled=False, project_id=90e1c2541a004e6c8dfbb78a1c3a35ac, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=90e1c2541a004e6c8dfbb78a1c3a35ac, updated_at=2018-06-30T01:56:35Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:409
2018-06-30 03:56:38.767 20577 DEBUG oslo_concurrency.lockutils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:56:38.769 20577 DEBUG oslo_concurrency.lockutils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:56:38.771 20577 INFO neutron.agent.dhcp.agent [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-30T01:56:34Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-639aee43-8461-496d-9fcf-c6db58e2fad0, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4f3a9de3-0882-42a8-b7e9-2407f70f728a', u'ip_address': u'11.135.1.2'}], id=7e2a3353-0146-469e-946a-d5421e490afd, mac_address=fa:16:3e:75:3c:79, name=, network_id=639aee43-8461-496d-9fcf-c6db58e2fad0, port_security_enabled=False, project_id=90e1c2541a004e6c8dfbb78a1c3a35ac, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=90e1c2541a004e6c8dfbb78a1c3a35ac, updated_at=2018-06-30T01:56:37Z
2018-06-30 03:56:38.775 20577 DEBUG neutron.agent.dhcp.agent [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 03:56:38.777 20577 DEBUG neutron.agent.linux.dhcp [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Building host file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 03:56:38.779 20577 DEBUG neutron.agent.linux.dhcp [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Done building host file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 03:56:38.784 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '29046'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:38.799 20577 DEBUG neutron.agent.linux.dhcp [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Reloading allocations for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 03:56:38.801 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:38.817 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-6', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:56:38.836 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:56:38.837 20577 DEBUG neutron.agent.linux.utils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:56:38.838 20577 DEBUG neutron.agent.linux.external_process [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 03:56:38.840 20577 DEBUG oslo_concurrency.lockutils [req-f9243144-f135-42fa-9819-2127097de071 - - - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:57:12.771 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 03:57:12.775 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 03:57:17.371 20577 DEBUG oslo_concurrency.lockutils [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:57:17.372 20577 INFO neutron.agent.dhcp.agent [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-30T01:57:15Z, description=, device_id=d4086585-5b99-4e8e-83d8-f3f7ffc1f6eb, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4f3a9de3-0882-42a8-b7e9-2407f70f728a', u'ip_address': u'11.135.1.1'}], id=3819e015-4530-4e8a-9d27-ee257bcd442e, mac_address=fa:16:3e:00:a6:5b, name=, network_id=639aee43-8461-496d-9fcf-c6db58e2fad0, port_security_enabled=False, project_id=90e1c2541a004e6c8dfbb78a1c3a35ac, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=90e1c2541a004e6c8dfbb78a1c3a35ac, updated_at=2018-06-30T01:57:16Z
2018-06-30 03:57:17.373 20577 DEBUG neutron.agent.dhcp.agent [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 03:57:17.375 20577 DEBUG neutron.agent.linux.dhcp [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Building host file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 03:57:17.377 20577 DEBUG neutron.agent.linux.dhcp [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Done building host file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 03:57:17.381 20577 DEBUG neutron.agent.linux.utils [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Running command (rootwrap daemon): ['kill', '-HUP', '29046'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:17.402 20577 DEBUG neutron.agent.linux.dhcp [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Reloading allocations for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 03:57:17.404 20577 DEBUG neutron.agent.linux.utils [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:17.419 20577 DEBUG neutron.agent.linux.utils [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-6', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:17.434 20577 DEBUG neutron.agent.linux.utils [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:57:17.435 20577 DEBUG neutron.agent.linux.utils [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:57:17.436 20577 DEBUG neutron.agent.linux.external_process [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 03:57:17.437 20577 DEBUG oslo_concurrency.lockutils [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:57:18.622 20577 DEBUG oslo_concurrency.lockutils [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:57:18.624 20577 INFO neutron.agent.dhcp.agent [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-30T01:57:15Z, description=, device_id=d4086585-5b99-4e8e-83d8-f3f7ffc1f6eb, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4f3a9de3-0882-42a8-b7e9-2407f70f728a', u'ip_address': u'11.135.1.1'}], id=3819e015-4530-4e8a-9d27-ee257bcd442e, mac_address=fa:16:3e:00:a6:5b, name=, network_id=639aee43-8461-496d-9fcf-c6db58e2fad0, port_security_enabled=False, project_id=90e1c2541a004e6c8dfbb78a1c3a35ac, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=90e1c2541a004e6c8dfbb78a1c3a35ac, updated_at=2018-06-30T01:57:16Z
2018-06-30 03:57:18.625 20577 DEBUG neutron.agent.dhcp.agent [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 03:57:18.627 20577 DEBUG neutron.agent.linux.dhcp [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Building host file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 03:57:18.629 20577 DEBUG neutron.agent.linux.dhcp [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Done building host file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 03:57:18.633 20577 DEBUG neutron.agent.linux.utils [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Running command (rootwrap daemon): ['kill', '-HUP', '29046'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:18.647 20577 DEBUG neutron.agent.linux.dhcp [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Reloading allocations for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 03:57:18.649 20577 DEBUG neutron.agent.linux.utils [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:18.662 20577 DEBUG neutron.agent.linux.utils [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-6', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:18.675 20577 DEBUG neutron.agent.linux.utils [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:57:18.676 20577 DEBUG neutron.agent.linux.utils [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:57:18.677 20577 DEBUG neutron.agent.linux.external_process [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 03:57:18.678 20577 DEBUG oslo_concurrency.lockutils [req-afeb3c30-f947-4207-ba3f-6f1ffd4526db a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:57:21.655 20577 DEBUG oslo_concurrency.lockutils [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:57:21.656 20577 INFO neutron.agent.dhcp.agent [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-30T01:57:15Z, description=, device_id=d4086585-5b99-4e8e-83d8-f3f7ffc1f6eb, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4f3a9de3-0882-42a8-b7e9-2407f70f728a', u'ip_address': u'11.135.1.1'}], id=3819e015-4530-4e8a-9d27-ee257bcd442e, mac_address=fa:16:3e:00:a6:5b, name=, network_id=639aee43-8461-496d-9fcf-c6db58e2fad0, port_security_enabled=False, project_id=90e1c2541a004e6c8dfbb78a1c3a35ac, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=90e1c2541a004e6c8dfbb78a1c3a35ac, updated_at=2018-06-30T01:57:17Z
2018-06-30 03:57:21.658 20577 DEBUG neutron.agent.dhcp.agent [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 03:57:21.660 20577 DEBUG neutron.agent.linux.dhcp [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Building host file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 03:57:21.662 20577 DEBUG neutron.agent.linux.dhcp [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Done building host file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 03:57:21.673 20577 DEBUG neutron.agent.linux.utils [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '29046'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:21.691 20577 DEBUG neutron.agent.linux.dhcp [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Reloading allocations for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 03:57:21.693 20577 DEBUG neutron.agent.linux.utils [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:21.707 20577 DEBUG neutron.agent.linux.utils [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-6', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:21.724 20577 DEBUG neutron.agent.linux.utils [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:57:21.727 20577 DEBUG neutron.agent.linux.utils [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:57:21.728 20577 DEBUG neutron.agent.linux.external_process [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 03:57:21.729 20577 DEBUG oslo_concurrency.lockutils [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:57:22.374 20577 DEBUG oslo_concurrency.lockutils [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:57:22.377 20577 INFO neutron.agent.dhcp.agent [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-30T01:57:15Z, description=, device_id=d4086585-5b99-4e8e-83d8-f3f7ffc1f6eb, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4f3a9de3-0882-42a8-b7e9-2407f70f728a', u'ip_address': u'11.135.1.1'}], id=3819e015-4530-4e8a-9d27-ee257bcd442e, mac_address=fa:16:3e:00:a6:5b, name=, network_id=639aee43-8461-496d-9fcf-c6db58e2fad0, port_security_enabled=False, project_id=90e1c2541a004e6c8dfbb78a1c3a35ac, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=90e1c2541a004e6c8dfbb78a1c3a35ac, updated_at=2018-06-30T01:57:21Z
2018-06-30 03:57:22.379 20577 DEBUG neutron.agent.dhcp.agent [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 03:57:22.381 20577 DEBUG neutron.agent.linux.dhcp [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Building host file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 03:57:22.384 20577 DEBUG neutron.agent.linux.dhcp [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Done building host file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 03:57:22.390 20577 DEBUG neutron.agent.linux.utils [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '29046'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:22.409 20577 DEBUG neutron.agent.linux.dhcp [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Reloading allocations for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 03:57:22.411 20577 DEBUG neutron.agent.linux.utils [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:22.433 20577 DEBUG neutron.agent.linux.utils [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-6', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:22.451 20577 DEBUG neutron.agent.linux.utils [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:57:22.452 20577 DEBUG neutron.agent.linux.utils [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:57:22.454 20577 DEBUG neutron.agent.linux.external_process [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 03:57:22.455 20577 DEBUG oslo_concurrency.lockutils [req-e8f04da1-f557-4d8e-9f36-9331f3b62837 - - - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:57:31.684 20577 DEBUG oslo_concurrency.lockutils [req-42864662-10be-4bb1-8367-1a6c95999efb a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Acquired semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:57:31.685 20577 DEBUG oslo_concurrency.lockutils [req-42864662-10be-4bb1-8367-1a6c95999efb a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Releasing semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:57:35.110 20577 DEBUG oslo_concurrency.lockutils [req-a8a1a56a-43e7-4c65-988c-46d115128fa4 - - - - -] Acquired semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:57:35.111 20577 DEBUG oslo_concurrency.lockutils [req-a8a1a56a-43e7-4c65-988c-46d115128fa4 - - - - -] Releasing semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:57:35.871 20577 DEBUG oslo_concurrency.lockutils [req-a8a1a56a-43e7-4c65-988c-46d115128fa4 - - - - -] Acquired semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:57:35.872 20577 DEBUG oslo_concurrency.lockutils [req-a8a1a56a-43e7-4c65-988c-46d115128fa4 - - - - -] Releasing semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:57:56.772 20577 DEBUG oslo_concurrency.lockutils [req-a2c6ffd3-d8fd-4039-bc26-3fa6fa9115cc a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:57:56.773 20577 INFO neutron.agent.dhcp.agent [req-a2c6ffd3-d8fd-4039-bc26-3fa6fa9115cc a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-30T01:57:55Z, description=, device_id=09e91190-e26f-4e54-9f2b-1672a2a8b002, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4f3a9de3-0882-42a8-b7e9-2407f70f728a', u'ip_address': u'11.135.1.5'}], id=fc128f30-1f15-4bfa-ae76-7a9a78c11453, mac_address=fa:16:3e:90:ef:e6, name=, network_id=639aee43-8461-496d-9fcf-c6db58e2fad0, port_security_enabled=True, project_id=90e1c2541a004e6c8dfbb78a1c3a35ac, revision_number=3, security_groups=[u'c2748690-a583-4ae6-a2c5-b1d4364c91aa'], status=DOWN, tags=[], tenant_id=90e1c2541a004e6c8dfbb78a1c3a35ac, updated_at=2018-06-30T01:57:55Z
2018-06-30 03:57:56.774 20577 DEBUG neutron.agent.dhcp.agent [req-a2c6ffd3-d8fd-4039-bc26-3fa6fa9115cc a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 03:57:56.775 20577 DEBUG neutron.agent.linux.dhcp [req-a2c6ffd3-d8fd-4039-bc26-3fa6fa9115cc a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Building host file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 03:57:56.776 20577 DEBUG neutron.agent.linux.dhcp [req-a2c6ffd3-d8fd-4039-bc26-3fa6fa9115cc a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Done building host file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 03:57:56.780 20577 DEBUG neutron.agent.linux.utils [req-a2c6ffd3-d8fd-4039-bc26-3fa6fa9115cc a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Running command (rootwrap daemon): ['kill', '-HUP', '29046'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:56.801 20577 DEBUG neutron.agent.linux.dhcp [req-a2c6ffd3-d8fd-4039-bc26-3fa6fa9115cc a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Reloading allocations for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 03:57:56.803 20577 DEBUG neutron.agent.linux.utils [req-a2c6ffd3-d8fd-4039-bc26-3fa6fa9115cc a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:56.818 20577 DEBUG neutron.agent.linux.utils [req-a2c6ffd3-d8fd-4039-bc26-3fa6fa9115cc a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-6', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:56.837 20577 DEBUG neutron.agent.linux.utils [req-a2c6ffd3-d8fd-4039-bc26-3fa6fa9115cc a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:57:56.838 20577 DEBUG neutron.agent.linux.utils [req-a2c6ffd3-d8fd-4039-bc26-3fa6fa9115cc a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:57:56.839 20577 DEBUG neutron.agent.linux.external_process [req-a2c6ffd3-d8fd-4039-bc26-3fa6fa9115cc a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 03:57:56.844 20577 DEBUG oslo_concurrency.lockutils [req-a2c6ffd3-d8fd-4039-bc26-3fa6fa9115cc a661f6701a074bb4a028410a6dd60660 90e1c2541a004e6c8dfbb78a1c3a35ac - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:57:59.419 20577 DEBUG oslo_concurrency.lockutils [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:57:59.420 20577 INFO neutron.agent.dhcp.agent [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-30T01:57:55Z, description=, device_id=09e91190-e26f-4e54-9f2b-1672a2a8b002, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4f3a9de3-0882-42a8-b7e9-2407f70f728a', u'ip_address': u'11.135.1.5'}], id=fc128f30-1f15-4bfa-ae76-7a9a78c11453, mac_address=fa:16:3e:90:ef:e6, name=, network_id=639aee43-8461-496d-9fcf-c6db58e2fad0, port_security_enabled=True, project_id=90e1c2541a004e6c8dfbb78a1c3a35ac, revision_number=4, security_groups=[u'c2748690-a583-4ae6-a2c5-b1d4364c91aa'], status=DOWN, tags=[], tenant_id=90e1c2541a004e6c8dfbb78a1c3a35ac, updated_at=2018-06-30T01:57:58Z
2018-06-30 03:57:59.422 20577 DEBUG neutron.agent.dhcp.agent [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 03:57:59.424 20577 DEBUG neutron.agent.linux.dhcp [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Building host file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 03:57:59.426 20577 DEBUG neutron.agent.linux.dhcp [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Done building host file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 03:57:59.431 20577 DEBUG neutron.agent.linux.utils [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '29046'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:59.445 20577 DEBUG neutron.agent.linux.dhcp [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Reloading allocations for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 03:57:59.447 20577 DEBUG neutron.agent.linux.utils [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:59.463 20577 DEBUG neutron.agent.linux.utils [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-6', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:57:59.482 20577 DEBUG neutron.agent.linux.utils [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:57:59.483 20577 DEBUG neutron.agent.linux.utils [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:57:59.484 20577 DEBUG neutron.agent.linux.external_process [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 03:57:59.485 20577 DEBUG oslo_concurrency.lockutils [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:58:00.192 20577 DEBUG oslo_concurrency.lockutils [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:58:00.193 20577 INFO neutron.agent.dhcp.agent [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-30T01:57:55Z, description=, device_id=09e91190-e26f-4e54-9f2b-1672a2a8b002, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4f3a9de3-0882-42a8-b7e9-2407f70f728a', u'ip_address': u'11.135.1.5'}], id=fc128f30-1f15-4bfa-ae76-7a9a78c11453, mac_address=fa:16:3e:90:ef:e6, name=, network_id=639aee43-8461-496d-9fcf-c6db58e2fad0, port_security_enabled=True, project_id=90e1c2541a004e6c8dfbb78a1c3a35ac, revision_number=5, security_groups=[u'c2748690-a583-4ae6-a2c5-b1d4364c91aa'], status=DOWN, tags=[], tenant_id=90e1c2541a004e6c8dfbb78a1c3a35ac, updated_at=2018-06-30T01:57:59Z
2018-06-30 03:58:00.195 20577 DEBUG neutron.agent.dhcp.agent [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 03:58:00.197 20577 DEBUG neutron.agent.linux.dhcp [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Building host file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 03:58:00.200 20577 DEBUG neutron.agent.linux.dhcp [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Done building host file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 03:58:00.205 20577 DEBUG neutron.agent.linux.utils [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '29046'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:58:00.219 20577 DEBUG neutron.agent.linux.dhcp [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Reloading allocations for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 03:58:00.221 20577 DEBUG neutron.agent.linux.utils [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:58:00.239 20577 DEBUG neutron.agent.linux.utils [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-6', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 03:58:00.257 20577 DEBUG neutron.agent.linux.utils [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:58:00.259 20577 DEBUG neutron.agent.linux.utils [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 03:58:00.260 20577 DEBUG neutron.agent.linux.external_process [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 03:58:00.261 20577 DEBUG oslo_concurrency.lockutils [req-f78ef13f-0927-4c4e-a99a-f515cf7ef9fd db7e024eefa34f0d9525ad1ee1475527 910acf3b84b64f358e48f506530f7cb1 - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:58:12.772 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 03:58:12.776 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 03:58:26.708 20577 DEBUG oslo_concurrency.lockutils [req-ec963c06-3453-4e64-99e9-90a4e0f1b4d1 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Acquired semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:58:26.709 20577 DEBUG oslo_concurrency.lockutils [req-ec963c06-3453-4e64-99e9-90a4e0f1b4d1 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Releasing semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:58:28.293 20577 DEBUG oslo_concurrency.lockutils [req-ec963c06-3453-4e64-99e9-90a4e0f1b4d1 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Acquired semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 03:58:28.294 20577 DEBUG oslo_concurrency.lockutils [req-ec963c06-3453-4e64-99e9-90a4e0f1b4d1 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Releasing semaphore "dhcp-agent-network-lock-bccef923-8280-488c-84aa-955d7841d8ff" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 03:59:12.772 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 03:59:12.776 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 04:00:12.773 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 04:00:12.778 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 04:01:12.773 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 04:01:12.778 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 04:02:12.773 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 04:02:12.777 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 04:02:50.216 20577 DEBUG oslo_concurrency.lockutils [req-d40c3e30-ab96-43cb-9e9e-00ee48e4030a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 04:02:50.218 20577 DEBUG neutron.agent.dhcp.agent [req-d40c3e30-ab96-43cb-9e9e-00ee48e4030a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 04:02:50.220 20577 DEBUG neutron.agent.linux.utils [req-d40c3e30-ab96-43cb-9e9e-00ee48e4030a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'dhcp_release', 'tap7e2a3353-01', '11.135.1.5', 'fa:16:3e:90:ef:e6'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:02:50.239 20577 DEBUG neutron.agent.linux.dhcp [req-d40c3e30-ab96-43cb-9e9e-00ee48e4030a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Building host file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 04:02:50.256 20577 DEBUG neutron.agent.linux.dhcp [req-d40c3e30-ab96-43cb-9e9e-00ee48e4030a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Done building host file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 04:02:50.279 20577 DEBUG neutron.agent.linux.utils [req-d40c3e30-ab96-43cb-9e9e-00ee48e4030a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '29046'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:02:50.289 20577 DEBUG neutron.agent.linux.dhcp [req-d40c3e30-ab96-43cb-9e9e-00ee48e4030a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Reloading allocations for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 04:02:50.291 20577 DEBUG neutron.agent.linux.utils [req-d40c3e30-ab96-43cb-9e9e-00ee48e4030a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:02:50.303 20577 DEBUG neutron.agent.linux.utils [req-d40c3e30-ab96-43cb-9e9e-00ee48e4030a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-6', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:02:50.314 20577 DEBUG neutron.agent.linux.utils [req-d40c3e30-ab96-43cb-9e9e-00ee48e4030a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:02:50.315 20577 DEBUG neutron.agent.linux.utils [req-d40c3e30-ab96-43cb-9e9e-00ee48e4030a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:02:50.315 20577 DEBUG neutron.agent.linux.external_process [req-d40c3e30-ab96-43cb-9e9e-00ee48e4030a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 04:02:50.316 20577 DEBUG oslo_concurrency.lockutils [req-d40c3e30-ab96-43cb-9e9e-00ee48e4030a 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 04:03:12.773 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 04:03:12.778 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 04:03:45.826 20577 DEBUG oslo_concurrency.lockutils [req-1cf8973e-e965-4c20-8394-dfd452380dc5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 04:03:45.827 20577 DEBUG neutron.agent.dhcp.agent [req-1cf8973e-e965-4c20-8394-dfd452380dc5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 04:03:45.830 20577 DEBUG neutron.agent.linux.utils [req-1cf8973e-e965-4c20-8394-dfd452380dc5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'dhcp_release', 'tap7e2a3353-01', '11.135.1.1', 'fa:16:3e:00:a6:5b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:03:45.854 20577 DEBUG neutron.agent.linux.dhcp [req-1cf8973e-e965-4c20-8394-dfd452380dc5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Building host file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 04:03:45.861 20577 DEBUG neutron.agent.linux.dhcp [req-1cf8973e-e965-4c20-8394-dfd452380dc5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Done building host file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 04:03:45.870 20577 DEBUG neutron.agent.linux.utils [req-1cf8973e-e965-4c20-8394-dfd452380dc5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '29046'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:03:45.883 20577 DEBUG neutron.agent.linux.dhcp [req-1cf8973e-e965-4c20-8394-dfd452380dc5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Reloading allocations for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 04:03:45.884 20577 DEBUG neutron.agent.linux.utils [req-1cf8973e-e965-4c20-8394-dfd452380dc5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:03:45.899 20577 DEBUG neutron.agent.linux.utils [req-1cf8973e-e965-4c20-8394-dfd452380dc5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-6', 'route', 'list', 'dev', 'tap7e2a3353-01'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:03:45.913 20577 DEBUG neutron.agent.linux.utils [req-1cf8973e-e965-4c20-8394-dfd452380dc5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:03:45.915 20577 DEBUG neutron.agent.linux.utils [req-1cf8973e-e965-4c20-8394-dfd452380dc5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:03:45.915 20577 DEBUG neutron.agent.linux.external_process [req-1cf8973e-e965-4c20-8394-dfd452380dc5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 04:03:45.916 20577 DEBUG oslo_concurrency.lockutils [req-1cf8973e-e965-4c20-8394-dfd452380dc5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 04:04:12.773 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 04:04:12.778 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-30 04:04:15.379 20577 DEBUG oslo_concurrency.lockutils [req-857fd016-85ae-4cdf-94e9-30082d9e6ec5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 04:04:15.381 20577 DEBUG neutron.agent.dhcp.agent [req-857fd016-85ae-4cdf-94e9-30082d9e6ec5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 04:04:15.383 20577 DEBUG neutron.agent.linux.utils [req-857fd016-85ae-4cdf-94e9-30082d9e6ec5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['kill', '-9', '29046'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:15.402 20577 DEBUG neutron.agent.linux.utils [req-857fd016-85ae-4cdf-94e9-30082d9e6ec5 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-30 04:04:15.455 20577 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tap7e2a3353-01) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-30 04:04:15.560 20577 DEBUG neutron.agent.linux.utils [req-3d27c8d5-d5f5-4879-9e76-07d0de2ced77 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:15.576 20577 DEBUG neutron.agent.linux.utils [req-3d27c8d5-d5f5-4879-9e76-07d0de2ced77 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:15.590 20577 DEBUG oslo_concurrency.lockutils [req-3d27c8d5-d5f5-4879-9e76-07d0de2ced77 - - - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 04:04:15.682 20577 DEBUG neutron.agent.dhcp.agent [req-f0582d13-c211-405c-a6e8-edcceab54411 - - - - -] resync (639aee43-8461-496d-9fcf-c6db58e2fad0): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:255
2018-06-30 04:04:15.683 20577 INFO neutron.agent.dhcp.agent [req-f0582d13-c211-405c-a6e8-edcceab54411 - - - - -] Synchronizing state
2018-06-30 04:04:16.484 20577 INFO neutron.agent.dhcp.agent [req-8a83ada4-e051-4a70-bdeb-430b784b522f - - - - -] All active networks have been fetched through RPC.
2018-06-30 04:04:16.486 20577 INFO neutron.agent.dhcp.agent [-] Starting network 639aee43-8461-496d-9fcf-c6db58e2fad0 dhcp configuration
2018-06-30 04:04:16.486 20577 DEBUG neutron.agent.dhcp.agent [-] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 04:04:16.488 20577 DEBUG neutron.agent.linux.utils [-] Unable to access /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:04:16.489 20577 DEBUG neutron.agent.linux.dhcp [-] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-639aee43-8461-496d-9fcf-c6db58e2fad0 on network 639aee43-8461-496d-9fcf-c6db58e2fad0 does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1267
2018-06-30 04:04:16.489 20577 DEBUG neutron.agent.linux.dhcp [-] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-639aee43-8461-496d-9fcf-c6db58e2fad0 on network 639aee43-8461-496d-9fcf-c6db58e2fad0 does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1288
2018-06-30 04:04:16.494 20577 INFO neutron.agent.dhcp.agent [-] Starting network bccef923-8280-488c-84aa-955d7841d8ff dhcp configuration
2018-06-30 04:04:16.494 20577 INFO neutron.agent.dhcp.agent [-] Finished network bccef923-8280-488c-84aa-955d7841d8ff dhcp configuration
2018-06-30 04:04:19.759 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:19.776 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:19.809 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:19.831 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:19.849 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:19.870 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', 'link', 'set', 'tapdccf3186-3c', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:19.899 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-o', 'link', 'show', 'tapdccf3186-3c'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:19.918 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-30 04:04:19.945 20577 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tapdccf3186-3c) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-30 04:04:19.948 20577 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-06-30 04:04:19.951 20577 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tapdccf3186-3c) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-30 04:04:19.953 20577 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'dccf3186-3c56-41d4-bd42-214b99dfc42d', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:a1:c5:00'})), record=tapdccf3186-3c) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-30 04:04:19.996 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tapdccf3186-3c', 'address', 'fa:16:3e:a1:c5:00'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.008 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.024 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tapdccf3186-3c', 'netns', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.055 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', 'link', 'set', 'tapdccf3186-3c', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.070 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', 'link', 'set', 'tapdccf3186-3c', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.091 20577 DEBUG oslo_concurrency.lockutils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Acquired semaphore "iptables-qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 04:04:20.092 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.115 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.140 20577 DEBUG neutron.agent.linux.iptables_manager [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-06-30 04:04:20.142 20577 DEBUG oslo_concurrency.lockutils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Releasing semaphore "iptables-qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 04:04:20.143 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-o', 'link', 'show', 'tapdccf3186-3c'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.162 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', 'addr', 'show', 'tapdccf3186-3c'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.183 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'addr', 'add', '11.135.1.2/24', 'scope', 'global', 'dev', 'tapdccf3186-3c', 'brd', '11.135.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.202 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'list', 'dev', 'tapdccf3186-3c'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.217 20577 DEBUG neutron.agent.linux.dhcp [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Setting IPv4 gateway for dhcp netns on net 639aee43-8461-496d-9fcf-c6db58e2fad0 to 11.135.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1164
2018-06-30 04:04:20.219 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.135.1.1', 'dev', 'tapdccf3186-3c'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.235 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-6', 'route', 'list', 'dev', 'tapdccf3186-3c'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.247 20577 DEBUG neutron.agent.linux.dhcp [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Cleaning stale devices for network 639aee43-8461-496d-9fcf-c6db58e2fad0 _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1369
2018-06-30 04:04:20.248 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.259 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.276 20577 DEBUG neutron.agent.linux.dhcp [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Building initial lease file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:623
2018-06-30 04:04:20.278 20577 DEBUG neutron.agent.linux.dhcp [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Done building initial lease file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/leases with contents:
1530410660 fa:16:3e:a1:c5:00 11.135.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:651
2018-06-30 04:04:20.279 20577 DEBUG neutron.agent.linux.dhcp [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Building host file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 04:04:20.281 20577 DEBUG neutron.agent.linux.dhcp [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Done building host file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 04:04:20.284 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Unable to access /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:04:20.285 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host', '--addn-hosts=/var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tapdccf3186-3c', '--dhcp-range=set:tag0,11.135.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.346 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:04:20.352 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:04:20.354 20577 DEBUG neutron.agent.linux.external_process [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 04:04:20.355 20577 INFO neutron.agent.dhcp.agent [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Finished network 639aee43-8461-496d-9fcf-c6db58e2fad0 dhcp configuration
2018-06-30 04:04:20.357 20577 INFO neutron.agent.dhcp.agent [req-8a83ada4-e051-4a70-bdeb-430b784b522f - - - - -] Synchronizing state complete
2018-06-30 04:04:20.359 20577 DEBUG oslo_concurrency.lockutils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 04:04:20.360 20577 INFO neutron.agent.dhcp.agent [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-30T02:04:17Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-639aee43-8461-496d-9fcf-c6db58e2fad0, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4f3a9de3-0882-42a8-b7e9-2407f70f728a', u'ip_address': u'11.135.1.2'}], id=dccf3186-3c56-41d4-bd42-214b99dfc42d, mac_address=fa:16:3e:a1:c5:00, name=, network_id=639aee43-8461-496d-9fcf-c6db58e2fad0, port_security_enabled=False, project_id=90e1c2541a004e6c8dfbb78a1c3a35ac, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=90e1c2541a004e6c8dfbb78a1c3a35ac, updated_at=2018-06-30T02:04:17Z
2018-06-30 04:04:20.361 20577 DEBUG neutron.agent.dhcp.agent [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 04:04:20.363 20577 DEBUG neutron.agent.linux.dhcp [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Building host file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 04:04:20.368 20577 DEBUG neutron.agent.linux.dhcp [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Done building host file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 04:04:20.372 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '32371'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.394 20577 DEBUG neutron.agent.linux.dhcp [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Reloading allocations for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 04:04:20.395 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'list', 'dev', 'tapdccf3186-3c'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.414 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-6', 'route', 'list', 'dev', 'tapdccf3186-3c'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.430 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:04:20.432 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:04:20.433 20577 DEBUG neutron.agent.linux.external_process [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 04:04:20.434 20577 DEBUG oslo_concurrency.lockutils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 04:04:20.436 20577 DEBUG oslo_concurrency.lockutils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 04:04:20.437 20577 INFO neutron.agent.dhcp.agent [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-30T02:04:17Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-639aee43-8461-496d-9fcf-c6db58e2fad0, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'4f3a9de3-0882-42a8-b7e9-2407f70f728a', u'ip_address': u'11.135.1.2'}], id=dccf3186-3c56-41d4-bd42-214b99dfc42d, mac_address=fa:16:3e:a1:c5:00, name=, network_id=639aee43-8461-496d-9fcf-c6db58e2fad0, port_security_enabled=False, project_id=90e1c2541a004e6c8dfbb78a1c3a35ac, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=90e1c2541a004e6c8dfbb78a1c3a35ac, updated_at=2018-06-30T02:04:19Z
2018-06-30 04:04:20.438 20577 DEBUG neutron.agent.dhcp.agent [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 04:04:20.440 20577 DEBUG neutron.agent.linux.dhcp [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Building host file: /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-30 04:04:20.442 20577 DEBUG neutron.agent.linux.dhcp [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Done building host file /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-30 04:04:20.447 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '32371'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.473 20577 DEBUG neutron.agent.linux.dhcp [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Reloading allocations for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-30 04:04:20.476 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-4', 'route', 'list', 'dev', 'tapdccf3186-3c'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.491 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0', 'ip', '-6', 'route', 'list', 'dev', 'tapdccf3186-3c'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:20.507 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:04:20.508 20577 DEBUG neutron.agent.linux.utils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:04:20.509 20577 DEBUG neutron.agent.linux.external_process [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 04:04:20.510 20577 DEBUG oslo_concurrency.lockutils [req-68de11a1-ef53-4c65-b76f-e297f3b15e52 - - - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 04:04:26.935 20577 DEBUG oslo_concurrency.lockutils [req-f459942e-5843-4063-9b50-3fa7bb34f2f2 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 04:04:26.936 20577 DEBUG neutron.agent.dhcp.agent [req-f459942e-5843-4063-9b50-3fa7bb34f2f2 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 04:04:26.940 20577 DEBUG neutron.agent.linux.utils [req-f459942e-5843-4063-9b50-3fa7bb34f2f2 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command (rootwrap daemon): ['kill', '-9', '32371'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:26.960 20577 DEBUG neutron.agent.linux.utils [req-f459942e-5843-4063-9b50-3fa7bb34f2f2 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-30 04:04:26.994 20577 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tapdccf3186-3c) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-30 04:04:27.079 20577 DEBUG neutron.agent.linux.utils [req-a5c06114-6c45-453c-9d8e-81ec4ae62bd5 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:27.092 20577 DEBUG neutron.agent.linux.utils [req-a5c06114-6c45-453c-9d8e-81ec4ae62bd5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:27.105 20577 DEBUG oslo_concurrency.lockutils [req-a5c06114-6c45-453c-9d8e-81ec4ae62bd5 - - - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 04:04:28.205 20577 DEBUG oslo_concurrency.lockutils [req-f459942e-5843-4063-9b50-3fa7bb34f2f2 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 04:04:28.633 20577 DEBUG neutron.agent.linux.utils [req-a6f40eb5-7345-4765-af85-34e9465d1529 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:04:28.635 20577 DEBUG neutron.agent.linux.utils [req-a6f40eb5-7345-4765-af85-34e9465d1529 - - - - -] Unable to access /var/lib/neutron/external/pids/639aee43-8461-496d-9fcf-c6db58e2fad0.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:04:28.636 20577 DEBUG neutron.agent.linux.external_process [req-a6f40eb5-7345-4765-af85-34e9465d1529 - - - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 04:04:28.638 20577 DEBUG neutron.agent.dhcp.agent [req-a6f40eb5-7345-4765-af85-34e9465d1529 - - - - -] Calling driver for network: 639aee43-8461-496d-9fcf-c6db58e2fad0 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-30 04:04:28.640 20577 DEBUG neutron.agent.linux.utils [req-a6f40eb5-7345-4765-af85-34e9465d1529 - - - - -] Unable to access /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:04:28.641 20577 DEBUG neutron.agent.linux.utils [req-a6f40eb5-7345-4765-af85-34e9465d1529 - - - - -] Unable to access /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-30 04:04:28.642 20577 DEBUG neutron.agent.linux.external_process [req-a6f40eb5-7345-4765-af85-34e9465d1529 - - - - -] No process started for 639aee43-8461-496d-9fcf-c6db58e2fad0 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-30 04:04:28.644 20577 DEBUG neutron.agent.linux.dhcp [req-a6f40eb5-7345-4765-af85-34e9465d1529 - - - - -] Impossibile accedere a /var/lib/neutron/dhcp/639aee43-8461-496d-9fcf-c6db58e2fad0/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:268
2018-06-30 04:04:28.645 20577 DEBUG neutron.agent.linux.dhcp [req-a6f40eb5-7345-4765-af85-34e9465d1529 - - - - -] No interface exists for network 639aee43-8461-496d-9fcf-c6db58e2fad0 destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1482
2018-06-30 04:04:28.690 20577 DEBUG neutron.agent.linux.utils [req-1b055c8c-b35e-4bb0-9b5c-f669627ec401 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-30 04:04:28.709 20577 DEBUG neutron.agent.linux.dhcp [req-1b055c8c-b35e-4bb0-9b5c-f669627ec401 - - - - -] Namespace already deleted: qdhcp-639aee43-8461-496d-9fcf-c6db58e2fad0 _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:246
2018-06-30 04:04:28.711 20577 DEBUG oslo_concurrency.lockutils [req-1b055c8c-b35e-4bb0-9b5c-f669627ec401 - - - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 04:04:29.974 20577 DEBUG oslo_concurrency.lockutils [req-f459942e-5843-4063-9b50-3fa7bb34f2f2 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Acquired semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-30 04:04:29.975 20577 DEBUG oslo_concurrency.lockutils [req-f459942e-5843-4063-9b50-3fa7bb34f2f2 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - - -] Releasing semaphore "dhcp-agent-network-lock-639aee43-8461-496d-9fcf-c6db58e2fad0" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-30 04:04:50.358 20577 DEBUG neutron.agent.dhcp.agent [req-8a83ada4-e051-4a70-bdeb-430b784b522f - - - - -] resync (639aee43-8461-496d-9fcf-c6db58e2fad0): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:255
2018-06-30 04:04:50.360 20577 INFO neutron.agent.dhcp.agent [req-8a83ada4-e051-4a70-bdeb-430b784b522f - - - - -] Synchronizing state
2018-06-30 04:04:51.278 20577 INFO neutron.agent.dhcp.agent [req-914bde5b-3148-47bc-842f-1d92f4e62ab8 - - - - -] All active networks have been fetched through RPC.
2018-06-30 04:04:51.280 20577 INFO neutron.agent.dhcp.agent [-] Starting network bccef923-8280-488c-84aa-955d7841d8ff dhcp configuration
2018-06-30 04:04:51.280 20577 INFO neutron.agent.dhcp.agent [-] Finished network bccef923-8280-488c-84aa-955d7841d8ff dhcp configuration
2018-06-30 04:04:51.281 20577 INFO neutron.agent.dhcp.agent [req-914bde5b-3148-47bc-842f-1d92f4e62ab8 - - - - -] Synchronizing state complete
2018-06-30 04:05:12.773 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-30 04:05:12.777 20577 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
